### ISSUE TITLE
QGEP SWMM raingages export

### ISSUE THREAD

**Issue by tproduit** [QGEP/QGEP#646]
Currently one raingage per subcatchment is exported. Then, the user can assign the same rain to each raingages. However, the most common case would a single raingage for the entire study area. 

In special cases, if the study area is very hilly and extended the user could then choose to add more raingages manually.

**Comment by sjib** [comment]
@DflGruBoe @kfischerar @urskaufmann What do you think about this. How many raingauges do you need in your projects?

**Comment by urskaufmann** [comment]
Till now the problem was, how many raingauges with good data are available in a study area. I have always defined the raindata in the hydraulic program, not in the vsa-dss database.

**Comment by tproduit** [comment]
Feedback from technical group:

- Literature shows that using several raingages provides more realistic results

- If raingages are inserted in QGEP (`measuring_point`):

    - the user can choose which rainage is exported
    -  or the catchments can be linked to nearest raingage

**Comment by ponceta** [comment]
@tproduit is the proposition added to the documentation?

@TriformMoritz do you agree with such strategy?

**Comment by tproduit** [comment]
Current export of rain gages works like this. It is documented in the [PR](https://github.com/QGEP/docs/pull/198)

QGEP_SWMM creates one raingage for each subcatchment. 

By default:

- Time serie: Each raingage is related to a default rain time serie called ``default_qgep_raingage_timeserie``. The time serie must be created and the value Series Name updated accordingly.
- Other parameters have default SWMM values

See ``vw_swmm_raingages.sql`` for details.

**Comment by ponceta** [comment]
To be closed as soon as doc PR is merged !

**Comment by TriformMoritz** [comment]
I usually use only one rain for the whole area for one typical rain event (with a given retourn time amd a typical curve). So I would leave it as default.
Various raingages are for interesting if you use real measurements of an event in an area or long time series (LTS), I would do this normally in SWMM (like Urs said).

So my recommandation: 
- Leave the default default_qgep_raingage_timeserie and link every catchment to the raingage (like it is done right now).
- Bonus 1: Make this "default rain" raingage visible in QGIS  (so you can modify it inside QGIS for all catchments).
- Bonus 2: Leave the option to create and link other raingages and export them.

So if I understand Timothee, this is how it will work!

**Comment by cymed** [comment]
I agree with @TriformMoritz. This also treats the bug that two raingages are created for a catchment if it is connected to fresh- and wastewater
